test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -6904,15 +6904,7 @@ BOOST_AUTO_TEST_CASE(function_types_sig)
 	CHECK_ERROR(text, TypeError, "Member \"selector\" not found");
 	text = R"(
 		contract C {
-			function f() view external returns (bytes4) {
```

Comment:
Why did we had a duplicate case here? :)

Perhaps something was lost during rebases.

Reply:
I think one of the cases used `var` before but we removed that and now they are identical.